reV/generation/generation.py

```diff
@@ -481,6 +501,61 @@ def _parse_nn_map(self):
 
         return nn_map
 
+    def _parse_bc(self, bias_correct):
+        """Parse the bias correction data.
+
+        Parameters
+        ----------
+        bias_correct : str | pd.DataFrame | None
+            Optional DataFrame or csv filepath to a wind or solar resource bias
+            correction table. This has columns: gid (can be index name), adder,
+            scalar. If both adder and scalar are present, the wind or solar
+            resource is corrected by (res*scalar)+adder. If either is not
+            present, scalar defaults to 1 and adder to 0. Only windspeed or
+            GHI+DNI are corrected depending on the technology. GHI and DNI are
+            corrected with the same correction factors.
+
+        Returns
+        -------
+        bias_correct : None | pd.DataFrame
+            None if not provided or extracted DataFrame with wind or solar
+            resource bias correction table. This has columns: gid (can be index
+            name), adder, scalar. If both adder and scalar are present, the
+            wind or solar resource is corrected by (res*scalar)+adder. If
+            either adder or scalar is not present, scalar defaults to 1 and
+            adder to 0. Only windspeed or GHI+DNI are corrected depending on
+            the technology. GHI and DNI are corrected with the same correction
+            factors.
+        """
+
+        if isinstance(bias_correct, type(None)):
```

Comment:
Ooo, genuinely curious - why `type(None)` here? Never seen this before, is it more robust?

Reply:
Haha great question, i think they're equivalent, i think i just used isinstance to match the syntax of the elif piece.